### DISCUSSION TITLE
【网站推荐】前端小白，写了一个小报童专栏精选导航站练手 xbt100.top

### DISCUSSION THREAD

**Discussion by lyqtzs** [just-talks/tech-talks#65] [3.分享创造]
推荐一个我自己写的小报童专栏导航网站：
[https://xbt100.top](https://xbt100.top)

目前精选收录了 200+ 个小报童专栏，涵盖了各个领域的专栏，包括副业、AI 、ChatGPT 、AI 绘画、自媒体、产品、职场、小红书、RPA 、公众号、搞钱、IP 、私域、电商、出海、互联网、编程、独立开发、沟通、读书、写作、心理、人际关系、成长、商业、工具、投资、web3 等。

以后每周会定期更新专栏数据。

[小报童专栏精选Top100排行榜](https://xbt100.top)

**Comment by adoyle-h** [comment]
不建议在本论坛分享内容聚合网站，因为内容太宽泛了。更不建议分享这种需要付费购买的专栏，因为这行为看起来像是在做推广。
你可以分享某篇具体的文章，谈谈你觉得好的部分。
或者你可以分享搭建这个网站用了哪些技术，遇到哪些有挑战的难点，分享你的经验。

**Comment by lyqtzs** [comment]
哦哦 这样子啊，明白了
---- 回复的原邮件 ----
发件人 ***@***.***> 日期 2024年07月01日 20:42 收件人 ***@***.***> 抄送至 ***@***.***>***@***.***> 主题 Re: [just-talks/tech-talks] 【网站推荐】前端小白，写了一个小报童专栏精选导航站练手 xbt100.top (Discussion #65) 
不建议在本论坛分享内容聚合网站，因为内容太宽泛了。更不建议分享这种需要付费购买的专栏，因为这行为看起来像是在做推广。
你可以分享某篇具体的文章，谈谈你觉得好的部分。
或者你可以分享搭建这个网站用了哪些技术，遇到哪些有挑战的难点，分享你的经验。

**Comment by kalxd** [comment]
上面的内容看得焦虑，能不能来轻松愉快的内容？

**Reply by NoahYau** [reply]
論壇都沒人了：（

**Reply by kalxd** [reply]
版主不给力，没有拉新人。

**Reply by adoyle-h** [reply]
版主在天上飞，暂时还没回到人间

**Reply by kalxd** [reply]
这是去了天上人间？

**Reply by NoahYau** [reply]
誰知道
